core/templates/app.constants.ts

```diff
@@ -20,6 +20,9 @@ import commonConstants from 'assets/constants';
 
 export const AppConstants = {
   ...commonConstants,
+
+  EndExploration: 'End Exploration',
```

Comment:
Constants should be upper-case and snake case.

Reply:
done